command.json

```diff
@@ -357,7 +364,7 @@
         },
         {
           "name": "provision",
-          "description": "Provision cluster on glcoud with Terraform",
+          "description": "Provision cluster on glcoud with Terraform (subcommands: <plan>, <apply>, <init> and <destroy>)",
```

Comment:
I thought the help shows the subcommands, or am I wrong? (not I'm not super happy with commander's help layout but it seems limited in how much you can change)

Reply:
These are "subsub" commands (e.g., provision plan) of `cloud`, and not (yet) defined in `command.json`.

At some point I like to add help examples of these subsub commands, but not sure where to put it yet.

Reply:
are they not working in the same way as other subcommands, for example `"commands"` inside "cloud"?

Reply:
They are just one level deeper than the other cloud commands, and I kept the same parameters as the rest with <environment> first. However, I think it would be better to change it too:

`cloud provision plan <environment>`,
`cloud provision apply <environment>`, etc

I'll do the same for `cloud secrets`.